services/web/env.conf

```diff
@@ -17,3 +17,6 @@ HTTP_BASIC_AUTH_PASS=
 
 # Sentry
 SENTRY_DSN=
+
+# GTM
```

Comment:
I don't think GTM is that familiar to everyone. Can we perhaps at least clarify the comment here:

```
# Google Tag Manager Analytics
```

Reply:
Good call